avocado/utils/asset.py

```diff
@@ -50,11 +50,11 @@ class Asset:
     Try to fetch/verify an asset file from multiple locations.
     """
 
-    def __init__(self, name, asset_hash=None, algorithm=None, locations=None,
-                 cache_dirs=None, expire=None, metadata=None):
+    def __init__(self, name=None, asset_hash=None, algorithm=None,
```

Comment:
Hi @beraldoleal, why are you not using `name=''` as default?

Reply:
Just to follow the same pattern on other arguments. In this case, it will be converted to '' anyway. But yes, it could be `name=''`